xija/limits.py

```diff
@@ -0,0 +1,54 @@
+def get_limit_spec(limit):
```

Comment:
Needs a license line.

Reply:
This function is not called anywhere here. Is there an idea that it would be useful in some other context? I don't mind leaving it, just asking.

Reply:
Added the license line. I originally had `get_limit_spec` used in `get_limit_color`, but now it is not anymore. I think it may be useful later.